sharness/t0030-simple-migration.sh

```diff
@@ -4,6 +4,8 @@ test_description="Simple fs-repo-migrations tests"
 
 . lib/test-lib.sh
 
+latestRepoVersion="12"
```

Comment:
This seems like a wacky change to have to do every time we do a release. Probably we could/should just hard code an immutable distpath so that the repo version is fixed, but at the moment we're just trying to get CI to be green.